Makefile

```diff
@@ -253,6 +256,14 @@ test-nightly-server: build-nightly-server
 check: check-shell-scripts check-yaml
 .PHONY: check
 
+
+
+# Format yaml files for yamllint standard
+yaml-fmt: $(filter  $(ALT_BIN)%,(YQ_CMD))
+	$(call yamls_reformat, $(CURDIR))
```

Comment:
Oh here it is. Honestly, I don't see the point of using call here. This is the only place YQ gets used so there's no need for code reuse.  I'd drop the define from the previous patch and just copy the YQ command to this line.